Tests/ClientTests/Coordinators/Mocks/ParentCoordinatorDelegateMock.swift

```diff
@@ -0,0 +1,14 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+@testable import Client
+
+class ParentCoordinatorDelegateMock: ParentCoordinatorDelegate {
```

Comment:
nit, preferrably having the Mock first so we know right away what this is about
```suggestion
class MockParentCoordinatorDelegate: ParentCoordinatorDelegate {
```